InvokePlaster.ps1

```diff
@@ -35,13 +35,17 @@ function Invoke-Plaster {
         $paramDictionary = New-Object System.Management.Automation.RuntimeDefinedParameterDictionary
 
         try {
-            $manifestPath = Join-Path $TemplatePath 'plasterManifest.xml'
-            $manifest = [xml](Get-Content $manifestPath -ErrorAction SilentlyContinue)
-
-            # The user-defined parameters in the Plaster manifest are converted to dynamic parameters
-            # which allows the user to provide all required parameters via the command line.
-            # This enables non-interactive use cases.
-            foreach ($node in $manifest.plasterManifest.parameters.ChildNodes) {
+            if($manifestPath -and 
```

Comment:
The extra test for the existence of manifest looks good.  However, can you clean up the coding style a bit?  Put a space between the `if` and the opening `(`.  Also indentation for this project is `4 spaces`.  But what I would do to avoid too much nesting is invert the logic of your test and return.